shell/plugins/dashboard-store/__tests__/utils/normalize-usecases.ts

```diff
@@ -0,0 +1,1018 @@
+export const usecases = {
```

Comment:
```suggestion
export const handleConflictUseCases = {
```

Reply:
Fixed. Will be updated once I push the commit